web/admin.py

```diff
@@ -211,8 +211,18 @@ class EndpointAdmin(admin.ModelAdmin):
     ordering = ('id',)
 
 
+class ReadOnlyAdmin(admin.ModelAdmin):
+    list_display = ['listing']
+
+    def has_add_permission(self, request, obj=None):
+        return False
+
+    def has_change_permission(self, request, obj=None):
+        return False
+
```

Comment:
I think you need to add `has_delete_permission`. At this point I can still delete exposures from the admin page.

<img width="554" alt="image" src="https://github.com/elixir-luxembourg/daisy/assets/29706637/17767d8b-f50e-4deb-a81d-fe985de0975b">

Reply:
i left it because i didn't see a problem in deleting an exposure from admin, do you think it should be removed?

Reply:
You are right, it should not cause any issue. I got stuck at 'read only' and did not think further.